client/src/components/footer/footer.js

```diff
@@ -185,17 +185,19 @@ class Footer extends Component {
             style={{ display: display ? "block" : "none" }}
           >
             <button
-              className="btn tw-bg-labLightGray btn-xl text-uppercase back "
+              //className="btn tw-bg-labLightGray btn-xl text-uppercase back "
+              className="back btn tw-w-32 tw-font-medium tw-bg-bgwhite text-uppercase  tw-border-b-labYellow tw-border-l-labYellow tw-border-8 tw-shadow-none"
               onClick={() => handleRedirect(actions, lab, body - 1)}
               style={{
                 display:
                   this.disappearBack(body) || hideOnLanding ? "none" : "block",
               }}
             >
-              Previous -{" "}
+              Back
+              {/* Previous -{" "}
```

Comment:
You can delete unused code, we can always look back at older commits if we need to get this code back